### DISCUSSION TITLE
Is pre-Normalization required for StructuredDataRegressor?

### DISCUSSION THREAD

**Discussion by Gorbov** [keras-team/autokeras#1612] [Q&A]
Is it required to do Normalization of features before using AutoKeras StructuredDataRegressor for train or AutoKeras StructuredDataRegressor takes care of it?

If necessary, where can I find a tutorial or some information on this topic like "best practice"?

**Comment by haifeng-jin** [comment]
No, normalization is a hyperparameter to tune during the search.
